english/explore-outside/journey-to-flaming-tower/jellybus/ask-person-about-destination/ask-person-about-mission/duel/duel.md

```diff
@@ -6,4 +6,8 @@ you in the least.
 to stop me and I'll prove that I'm the most
 powerful warrior in Chocosnackland!", you shout.
 
-He instantly leaps to his feet and lets out a roar.
\ No newline at end of file
+He instantly leaps to his feet and lets out a roar.
+
+Do you? 
+
+[Attack!](../attack/attack.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.